### PR TITLE
Add Github workflow and linters

### PR DESCRIPTION
**Pull request by blessedjasonmwanza**
## Add GitHub workflow and linters
- Add Github workflows
- Add linters

### CODE REVIEW

**Comment by blessedjasonmwanza** [comment]
Hi @blessedjasonmwanza , 

Great work on successfully isolating your work into branches 👏 
You've done well implementing Github flow and linters 🤓 , but there are still some which aren't addressed yet.


 ## Required Changes ♻️
It would be amazing if you can:
- [x] **Fix linter errors** (Check the details section to find out where the problems are in your code)
- [x] **Fix Webhint errors raised** - (Check the details section to find out where the problems are in your code as well)

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -11,24 +11,24 @@
     <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
```
Comment (blessedjasonmwanza) [inline index.html:12]: - A 'viewport' meta element was not specified. (meta-viewport)
> The viewport meta tag allows you to tell the mobile browser what size this virtual viewport should be. This is often useful if you're not actually changing your site's design for mobile, and it renders better with a larger or smaller virtual viewport

Here is [an insightful](https://developer.mozilla.org/en-US/docs/Web/HTML/Viewport_meta_tag) read encase of need ℹ️ .

File: index.html [hunk 2]
```diff
@@ -191,14 +191,14 @@
         <p>To execute multiple statements, use a block statement ({ ... }) to group those statements.</p>
         <p>Example:</p>
         <p>The following while loop iterates as long as n is less than three:</p>
-        <code>var n = 0; var x = 0; while (n < 3) { n++; x += n; }</code>
+        <code>var n = 0; var x = 0; while (n &lt; 3) { n++; x += n; }</code>
         <p>With each iteration, the loop increments n and adds that value to x. Therefore, x and n take on the following values:</p>
         <ul>
```
Comment (blessedjasonmwanza) [inline index.html:195]: Another little thing to change  😃 
- **Element “ul” not allowed as child of element “ul” in this context.** You'll notice that you have <ul> elements wrapped inside other ul elements in some sections of your code (Check _Data types_ section), something that needs just a little tweaking to meet coding standards.
> You can find some insightful methods on how to use best practices for these [here](https://www.w3schools.com/html/html_lists.asp) or from [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/li)

File: index.html [hunk 3]
```diff
@@ -217,7 +217,7 @@
       <section class="main-section" id="reference">
         <header>Reference</header>
```
Comment (blessedjasonmwanza) [inline index.html]: For your final touches:
- Consider replacing ```<header> ``` tags inside sections with ```<h1>``` through to ```<h6> ```. You can choose your range here 🤓 .

> Read up more on the importance of these (here)[https://www.w3.org/WAI/tutorials/page-structure/headings/] or on [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements)

**Comment by blessedjasonmwanza** [comment]
Good job so far on your progress to finish this project! You're almost done.

So close gif image

>Please resolve the suggested changes above and there should be nothing in your way to getting approved 🥇 .

All the best!

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @blessedjasonmwanza  🔥 

You are reviewing like a pro! Nicely done👍🏽 

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**